components/org.wso2.carbon.identity.verification.onfido.api/org.wso2.carbon.identity.verification.onfido.api.common/src/main/java/org/wso2/carbon/identity/verification/onfido/api/common/Constants.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright (c) 2023, WSO2 LLC. (http://www.wso2.com).
```

Comment:
Let's update the license headers

Reply:
Resolved via https://github.com/wso2-extensions/identity-verification-onfido/pull/12/commits/6f73cff0b813bfcfffbd6cfbe9d730df1f5fe3ed